draft-ietf-quic-multipath.md

```diff
@@ -1382,6 +1382,31 @@ Maximum Path Identifier:
   Receipt of a value that is higher than the local maximum value MUST
   be treated as a connection error of type PROTOCOL_VIOLATION.
 
+
+## PATH_CID_BLOCKED frames {#path-cid-blocked-frame}
+
+A sender SHOULD send a PATH_CID_BLOCKED frame (type=0x15228c0e) when
```

Comment:
I think we should actually change this "SHOULD" to "can". There is no requirement in interoperability that require the sender to send this. Or if we want normative language we should use "MAY".

The same comment of course applies to the PATHS_BLOCKED frame of course. Sorry I overlooked this earlier.